README.md

```diff
@@ -37,4 +39,36 @@ Steps of converting:
    Flags for location, plans, OS images are optional. 
    If you omit them empty and script can't find imported entities, 
    then product allows client to select necessary options manually during checkout.
-   We recommend you to check location, plan and OS image of converted products in Hostbill.
\ No newline at end of file
+   We recommend you to check location, plan and OS image of converted products in Hostbill.
+
+Step-by-step converting:
```

Comment:
Step-by-step -> Partial

Add a description for this conversion, something like: this conversion duplicates SolusVM 1 Hostbill order pages and converts them to SolusVM 2. SolusVM 1 accounts are converted into SolusVM 2 accounts if related SolusVM 1 servers were imported into SolusVM 2. It is possible to revert converted SolusVM 2 account back to SolusVM 1.